src/cascadia/TerminalApp/CommandPalette.xaml

```diff
@@ -423,7 +422,6 @@ the MIT License. See LICENSE in the project root for license information. -->
                 AllowDrop="False"
                 IsItemClickEnabled="True"
                 ItemClick="_listItemClicked"
-                PreviewKeyDown="_keyDownHandler"
```

Comment:
Just double-checking. So if we have focus on a list view item, any key stroke we make will go to the `UserControl`'s `PreviewKeyDown` handler on line 18? And that'll properly handle navigating the list view with the arrow keys and home/end?

Reply:
Yes. Actually Preview is going top down... and if the key is handled by the _keyDownHandler the propagation stops. In other words, this line was doing nothing functional :smile: